R/validate_config.R

```diff
@@ -233,6 +235,18 @@ val_round <- function(round, round_i, schema) {
         schema = schema
       )
     ),
+    purrr::imap(
+      model_task_grps,
+      \(.x, .y) {
+        validate_mt_round_id_pattern(
+          model_task_grp = .x, model_task_i = .y,
+          round_i = round_i,
+          schema = schema,
+          round_id_from_variable = round_id_from_variable,
+          round_id_var = round_id_var
+        )
+      }
```

Comment:
really minor point -- I notice that you used a different format here than in the calls above, where you had something more like the below:
```suggestion
      ~ validate_mt_round_id_pattern(
        model_task_grp = .x, model_task_i = .y,
        round_i = round_i,
        schema = schema,
        round_id_from_variable = round_id_from_variable,
        round_id_var = round_id_var
      )
```